kafka-streams-quickstart/aggregator/pom.xml

```diff
@@ -19,6 +19,7 @@
     <quarkus.platform.version>999-SNAPSHOT</quarkus.platform.version>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    <debezium.version>1.1.0.Final</debezium.version>
```

Comment:
I had to manage debezium.version as it was moved to quarkus-test-bom, but it seems quickstarts can't directly reference a quarkus version so it stays compatible with the quarkus-platform ?

Reply:
Let's ask @gsmet as he's more savvy with all those build details.

Reply:
I think it's OK as it is.